portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,23 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>Mauro Guerrero's Portfolio</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+	<div id="content">
+		<h1>Mauro Guerrero's Portfolio<img class="profile_image" src="images/profile_pic.jpg" alt="Avatar"/>
+			</h1>
+			<p>Hi, I am a Texas A&ampM Engineering Student that is currently pursuing a B.A. in Computer
+				Engineering.</p>
+			<p>Click here to get a random message:</p>
+			<button onclick="addRandomQuote()">Hello</button>
```

Comment:
Keep JavaScript contained to scripts (script.js in this case): https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

The [`addEventListener`](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener) method will be helpful here. You'll also need to grab a reference to this specific button in your JS to attach the listener (something like [`getElementById`](https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementById) should work).